### PR TITLE
Enables prow plugins for CAPV repo

### PR DESCRIPTION
**Pull request by srm09**

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -473,6 +473,10 @@ milestone_applier:
     release-1.0: v1.0.0
     release-0.4: v0.4
     release-0.3: v0.3
+  kubernetes-sigs/cluster-api-provider-vsphere:
+    main: v1.5
+    release-1.4: v1.4
```
Comment (sbueringer) [inline config/prow/plugins.yaml:478]: I assume v1.4 will be released soon?

**Comment by dims** [comment]
/approve
/hold

(please remove hold when ready!)

File: config/prow/plugins.yaml (hunk 1 above)
Comment (srm09) [reply config/prow/plugins.yaml:478]: Yes that's the plan :)

**Comment by srm09** [comment]
/unhold
since 1.4 was released yesterday and main points to 1.5 now

**Pull request merged**